client/api.go

```diff
@@ -35,119 +37,99 @@ import (
-	// CountrySpec is the client used to invoke /country_specs APIs.
-	// For more details see https://stripe.com/docs/api#country_specs.
-	CountrySpec *countryspec.Client
-	// Balance is the client used to invoke /balance and transaction-related APIs.
-	// For more details see https://stripe.com/docs/api#balance.
-	Balance *balance.Client
-	// EphemeralKeys is the client used to invoke /ephemeral_keys APIs.
-	// For more details see https://stripe.com/docs/api#ephemeral_keys.
-	EphemeralKeys *ephemeralkey.Client
-	// Events is the client used to invoke /events APIs.
-	// For more details see https://stripe.com/docs/api#events.
-	Events *event.Client
-	// Tokens is the client used to invoke /tokens APIs.
-	// For more details see https://stripe.com/docs/api#tokens.
-	Tokens *token.Client
-	// FileUploads is the client used to invoke the uploads /files APIs.
-	// For more details see https://stripe.com/docs/api#file_uploads.
-	FileUploads *fileupload.Client
-	// BitcoinReceivers is the client used to invoke /bitcoin/receivers APIs.
-	// For more details see https://stripe.com/docs/api#bitcoin_receivers.
-	BitcoinReceivers *bitcoinreceiver.Client
-	// BitcoinTransactions is the client used to invoke /bitcoin/transactions APIs.
-	// For more details see https://stripe.com/docs/api#bitcoin_receivers.
-	BitcoinTransactions *bitcointransaction.Client
 	// Reversals is the client used to invoke /transfers/reversals APIs.
 	Reversals *reversal.Client
-	// BankAccounts is the client used to invoke /accounts/bank_accounts APIs.
-	BankAccounts *bankaccount.Client
-	// Products is the client used to invoke /products APIs.
-	// For more details see https://stripe.com/docs/api#products.
-	Products *product.Client
-	// Orders is the client used to invoke /orders APIs.
-	// For more details see https://stripe.com/docs/api#orders.
-	Orders *order.Client
-	// OrderReturns is the client used to invoke /order_returns APIs.
-	// For more details, see https://stripe.com/docs/api#order_returns.
-	OrderReturns *orderreturn.Client
 	// Skus is the client used to invoke /skus APIs.
-	// For more details see https://stripe.com/docs/api#skus.
 	Skus *sku.Client
 	// Sources is the client used to invoke /sources APIs.
-	// For more details see https://stripe.com/docs/api#sources.
 	Sources *source.Client
-	// PaymentSource is used to invoke /sources APIs.
-	// For more details see https://stripe.com/docs/api.
-	PaymentSource *paymentsource.Client
-	// ExchangeRates is the client used to invoke /exchange_rates APIs.
-	ExchangeRates *exchangerate.Client
+	// SourceTransactions is the client used to invoke source transaction related APIs.
+	SourceTransactions *sourcetransaction.Client
+	// Subscriptions is the client used to invoke /subscriptions APIs.
+	Subscriptions *sub.Client
+	// SubscriptionItems is the client used to invoke subscription's items related APIs.
+	SubscriptionItems *subitem.Client
+	// Tokens is the client used to invoke /tokens APIs.
+	Tokens *token.Client
```

Comment:
I did not add `threedsecure` since it's been deprecated for well over a year. Is it the right call?

Reply:
Yeah, if no one's asked for it by now, let's just leave it out.